src/targets/gpu/fuse_ops.cpp

```diff
@@ -697,52 +697,6 @@ struct find_conv_bias_relu
         apply_conv_bias<miopen_conv_bias_relu>(*ctx, p, std::move(r));
     }
 };
-
-struct find_gemm_add
```

Comment:
@pfultz2  This one relies on setting `beta` value of 1 for the `dot` operator. That's why i'd removed. 

I can add beta field to `rocblas_gemm` and use it.